templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -19,6 +19,7 @@
     <jackson.version>$[jackson.version]</jackson.version> <!-- Overriding Jackson for fixing vulnerabilities -->
     <guava.version>$[guava.version]</guava.version>
     <devonfw.test.excluded.groups>system</devonfw.test.excluded.groups>
+	 <app.version>${revision}${changelist}</app.version>
```

Comment:
Excellent, this is exactly what we wanted to archive.
Please fix indendation and we can merge.